src/psyclone/psyGen.py

```diff
@@ -4583,6 +4786,117 @@ def coloured_text(self):
         return colored(self._clause_type, SCHEDULE_COLOUR_MAP["If"])
 
 
+class ACCKernelsDirective(ACCDirective):
+    '''
+    Class representing the !$ACC KERNELS directive in the PSyIR.
+
+    :param children: the PSyIR nodes to be enclosed in the Kernels region \
+                     and which are therefore children of this node.
+    :type children: list of sub-classes of :py:class:`psyclone.psyGen.Node`
+    :param parent: the parent of this node in the PSyIR.
+    :type parent: sub-class of :py:class:`psyclone.psyGen.Node`
+    :param bool default_present: whether or not to add the "default(present)" \
+                                 clause to the kernels directive.
+
+    :raises NotImplementedError: if default_present is False.
+
+    '''
+    def __init__(self, children=[], parent=None, default_present=False):
```

Comment:
W:4804, 4: Dangerous default value [] as argument (dangerous-default-value)

Reply:
Fixed. I've checked all the way back up to the Node base class and the default value for `children` is always None so we're OK.